_posts/2024-08-02-gsoc2024_talagayev.md

```diff
@@ -0,0 +1,193 @@
+which by selection would highlight the selected pharmacophores, thus by selecting a hydrophobic feature, the atoms of the molecule that are hydrophobic via the `RDKit` pharmacophore
+recognition were highlighted in the assigned colors.
+
+Here is a short summary of features that I added for the visualization, which would show a value for the corresponding feature or adjust the figure of the molecule to display the values:
+
+- **Phyisochemical Properties**, **Atom Indices**, **Partial Charges**, **Hydrogen Bond Donors&Acceptors**
+
+And here is a short summary of features, that will highlight a specific atoms and bonds corresponding to their features if they are selected:
+
+- **Rotatable Bonds**, **Partial Charge Heatmap**, **Functional Groups**, **Stereocenters**, **Murcko Scaffold**, **Pharmacophores**
+
+Here I would also address that for the pharmacophores there is a separate checkbox for each of the possible pharmacophore features, thus you can select the specific pharmaocphore feature
+that needs to be highlighted.
+
+For the functional groups it currently uses SMART Patterns to identify certain functional groups and will then generate the checkboxes of the functional groups that were recognized and
+by selecting them you are able to see the atoms of this functional group.
+
+### Code Restructure
+
+This is also a section that was done throughout the whole project since in the beginning, the code was mainly consisting of functions in one file and this was one of the points that I was
+glad that my mentors helped me with since the code restructure allowed me to go from a code that was first based upon only function to structure it with classes and then in the end the
+separate classes were moved into separate files, thus a clear structure of the code was created, where every file was distinctive from the other and had an important purpose and a clear
+structure 
+
+### Actions, Testing and Documentation
+
+This would be the final part of what I did and here I would put all these three parts together. Pytests were created for this project in addition to the code being uploaded to
+[Codecov](https://app.codecov.io/gh/talagayev/mdonatello) to display the coverage of the code. Furthermore, a CI CD was created, which installs the package and tests the pytests on
+Linux, MacOS and Windows systems with the Python versions 3.10, 3.11 and 3.12 with different `IpyWidgets` versions, with the oldest supported version for this code being `7.6.4`. Finally,
+a documentation based on the MDAKits Template was created and can be found here: https://mdonatello.readthedocs.io/
+
+## Current State
+
+The current state of MDonatello is, that it is available and working, thus it can be easily installed and used on your system. Some improvements can be made, but overall
+the current version is stable and applicable.
```

Comment:
I'd remove that sentence, `stable` has a specific meaning (the API and UI won't change much), but because this is the first release and you're still likely going to move things around, rename classes/functions...etc. this wouldn't really qualify as stable IMO.

Reply:
Yes makes sense, then I will remove it, somehow I though it meant that it runs without any crashes or errors.